CPT331.iOS/CPT331.iOS/EventCategory.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  EventCategory.swift
+//  CPT331.iOS
+//
+//  Created by Peter Weller on 8/10/2016.
+//  Copyright © 2016 Peter Weller. All rights reserved.
+//
+
+import UIKit
+
+enum EventCategory {
```

Comment:
We'll leave it for now, but I've added a task in Trello to look at a later date. 

Storing the category info (icon urls/marker colour, name, sub categories, ect) in a database means that the management of categories can be done without needing to update code. This is preferable in a commercial system, as coding is expensive 😑.